src/components/layer-controller/ChannelController.js

```diff
@@ -17,15 +17,13 @@ const toRgb = (on, arr) => {
 };
 
 function ChannelSelectionDropdown({
-  selectionIndex,
```

Comment:
This wasn't actually doing anything.

Reply:
With `native` on, it seems that we don't need this: https://material-ui.com/api/select/#props